gettsim/parameters/ges_rente.yaml

```diff
@@ -726,34 +726,84 @@ zugangsfaktor_veränderung_pro_jahr:
     en:
       Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
       claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
-      the pension claim is higher or lower.
+      the pension claim is higher or lower, unless special conditions apply.
   reference: § 35 Satz 2 SGB VI
   note:
     de: >-
       Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
-      implementiert.
+      implementiert. Bis einschließlich Jahrgang 1946 ist NRA 65, danach steigt es bis Jahrgang 1964.
+      Ab einschließlich Jahrgang 1964 gilt NRA 67. (todo jahrgänge vor 1946)
     en:
   type: piecewise_linear
   2002-01-01:
     0:
       lower_threshold: -inf
-      upper_threshold: 1947
+      upper_threshold: 1946
       rate_linear: 0
       intercept_at_lower_threshold: 65
     1:
-      lower_threshold: 1947
-      upper_threshold: 1971
+      upper_threshold: 1958
       rate_linear: 0.083333333
     2:
-      lower_threshold: 1971
+      upper_threshold: 1964
+      rate_linear: 0.16666666
+    3:
+      upper_threshold: inf
+      rate_linear: 0
+
+altersrente_für_frauen:
+  name:
+    de: Eintrittsalter für Altersrente für Frauen
+    en: Entry age for receiving Altersrente für Frauen
+  description:
+    de:
+      Stückweise lineare Funktion, die die Altersgrenzen angibt, ab denen Frauen geboren vor 1952 in rente gehen konnten.
+    en:
+      Piecewise linear function returning the age thresholds for retirement of women born before 1952.
+  reference: § 237a SGB VI
+  note:
+    de: >-
+      Frauen, die vor 1940 geboren wurden, konnten mit 60 abschlagsfrei in Rente gehen,
+      nach einer Wartezeit von 15 Jahren und mit mindestens 10 Beitragsjahren ab dem Alter von 40.
+      Ab 1940 wurde das Eintrittsalter sukzessive erhöhrt um einen Monat pro Geburtsmonat auf das Alter 65.
+      Für Kohorten 1945-1951 beträgt es konstant 65. Für Kohorten ab 1952 ist es abgeschafft. Ab dort regelaltersgrenze implementiert.
+    en:
+      Women born before 1940 were able to retire at age 60 without deductions (required waiting period of 15 years and 10 years of
+      contributions after age 40). The age threshold was increase by one month for each birthmonth up to age 65 for cohort 1945.
+      Between cohort 1945 and 1951 the threshold remained constant.
+      The special pension scheme for women was abolished for cohorts born afer 1952. regelaltersgrenze afterwards.
+  type: piecewise_linear
+  1980-01-01:
+    0:
+      lower_threshold: -inf
+      upper_threshold: 1939.916667
+      rate_linear: 0
+      intercept_at_lower_threshold: 60
+    1:
+      upper_threshold: 1944.916666666
+      rate_linear: 1
+    2:
+      upper_threshold: 1951.916666666 # constant 65
+      rate_linear: 0
+    3:
+      upper_threshold: 1952.0000 # since 1952 abolished -  regelaltersgrenze applies - jump in theshold by 6 month
+      rate_linear: 6
+    4:
```

Comment:
This is just a repetion of the parameters specified in `regelaltersgrenze`, isn't it? I think it would be nicer to not repeat them, but just calculate `altersrente_für_frauen` only for the kohorts for which it actually exists. (Or would you rather say it still exists, but has the similar values as the usual pension?)

Reply:
Yes, thats right. I realized this problem too. I tried to only define the thresholds until they existed, but the tests required a full definition of the thresholds for the `piecewise_polynomial` function up to "inf".

Reply:
option 3 rather not. if we keep the threshold constant this would suggest that they could retire earlier than they actually can. we could set it to 99 or something so that it is certainly higher than any other threshold.